src/util/support.js

```diff
@@ -1,5 +1,36 @@
+import isNativeRegex from 'regex-native-function'; // checks if a function is native using regex
```

Comment:
Yea, we were trying to make sure that the check wont return true if a polyfill had set the property already. Unless there's another reason not to use the regex way?